client/my-sites/checkout/checkout-thank-you/style.scss

```diff
@@ -38,7 +38,7 @@
 
 	&:last-child {
 		border-bottom: 1px solid var( --color-neutral-0 );
-		border-radius: 0 0 3px 3px;
+		border-radius: 0 0 2px 2px;
```

Comment:
This was forced by the linter. But makes sense, cause the `first-child` line above sets the border radius to 2px.